packages/react-admin/src/mui/form/SimpleForm.js

```diff
@@ -11,7 +11,9 @@ const formStyle = { padding: '0 1em 1em 1em' };
 
 export class SimpleForm extends Component {
     handleSubmitWithRedirect = (redirect = this.props.redirect) =>
-        this.props.handleSubmit(values => this.props.save(values, redirect));
+        this.props.handleSubmit((values, dispatch) =>
```

Comment:
I can find any reference to `dispatch` in redux-form. Where does it come from ?

Reply:
https://github.com/erikras/redux-form/blob/6b69da594c369bcdc8acd7a14f0af605c0abbb8d/src/handleSubmit.js#L39

Reply:
Even better: https://redux-form.com/7.2.0/docs/api/reduxform.md/#-code-onsubmit-function-code-optional-

I was afraid it was not documented